### PR TITLE
feat(what is Bonita): add ACM capabilities

### PR DESCRIPTION
**Pull request by NathalieC**
Explain ACM capabilities from 7.10 on.

### CODE REVIEW

File: md/what-is-bonita.md [hunk 1]
```diff
@@ -6,9 +6,23 @@ If you can't wait to start creating your first application with Bonita, you can
 
 ## What can I do with Bonita?
 
-Bonita is a Digital Process Automation platform that allows you to create applications based on processes. Processes can be  partially automated (if involving users) or fully automated.
+Bonita is a Digital Process Automation platform that allows you to create applications based on processes. Processes can be partially automated (if involving users) or fully automated.
```
Comment (rbioteau) [inline md/what-is-bonita.md]:
```suggestion
Bonita is a Digital Process Automation (DPA) platform that allows you to create applications based on processes. Processes can be partially automated (if involving users) or fully automated.
```

File: md/what-is-bonita.md [hunk 2]
```diff
@@ -6,9 +6,23 @@ If you can't wait to start creating your first application with Bonita, you can
...
+
+DPA, available with all Bonita editions, is applicable to a wide range of use cases: applications can go from loan request management, to student application processes,to analytical processes in drug discovery - and more! You can find the various industries where Bonita is used as well as use cases from our [website dedicated page](https://www.bonitasoft.com/industries).
+
+Starting with Bonita 7.10, you can also handle Adaptive Case Management with Bonita. 
+ACM (Adaptive or Advanced Case Management) is like DPA in the sense that it allows to handle cases with some automation. But it also relies on strong principles that make it different from DPA:
```
Comment (rbioteau) [inline md/what-is-bonita.md:14]: Advanced  ?

File: md/what-is-bonita.md [hunk 3]
```diff
@@ -6,9 +6,23 @@ If you can't wait to start creating your first application with Bonita, you can
...
+  - Empower the actors: 
+  Actors in charge of the case have some autonomy to lead the case to its closure: not all process paths are pre-determined. Some tasks are optional, some are discretionnary. Actors' specific knowledge make them fully able to choose the right next action for a case. 
+  - Expect the unexpected: 
+  A case can be stopped, resumed and can even change its state at any time without a path being predefined. New tasks may need to be created on the fly to reflect the specificity of a case while still benefiting from traceability.
+  - Adapt to the context: case type and state define the availability of tasks following a set of business rules. Those rules must be editable on the fly.
+  - Center on data: Case execution is not centered around a strict process, but around case data to create and update.
+
+ACM, available in Teamwork, Efficiency, Performance and Enterprise editions, is applicable to use cases including problem investigations  (like insurance claims), audits or decision making procedures.
+
+To learn more about how to implement ACM capabilities in Bonita, check the [documentation](use-bonita-acm.md).
```
Comment (rbioteau) [inline md/what-is-bonita.md]: use-bonita-acm.md is just the how to enable the event handler. It is by no mean a complete implementation guide.

**Review by rbioteau — CHANGES_REQUESTED** [review]

**Comment by rbioteau** [comment]
We should talk about the acm example at some point

File: md/what-is-bonita.md (hunk 3 above)
Comment (NathalieC) [reply md/what-is-bonita.md]: you're right. Changing that.

File: md/what-is-bonita.md (hunk 2 above)
Comment (NathalieC) [reply md/what-is-bonita.md:14]: Philippe's call.

**Review by NathalieC — COMMENTED** [review]
I've added it in another PR I think: the how to ACM PR.

File: md/what-is-bonita.md (hunk 3 above)
Comment (NathalieC) [reply md/what-is-bonita.md]: I've just put "use" and not "implement". This points to the documentation page where the ACM example will be presented.

**Pull request merged**